cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -411,6 +411,24 @@ public void getAll(@NotNull Context ctx) {
             String begin = ctx.queryParam(BEGIN);
             String end = ctx.queryParam(END);
             String timezone = ctx.queryParamAsClass(TIMEZONE, String.class).getOrDefault("UTC");
+            String versionDateParam = ctx.queryParam(VERSION_DATE);
+            ZonedDateTime versionDate = null;
+            if (versionDateParam != null) {
+                versionDate = DateUtils.parseUserDate(versionDateParam, timezone);
+            }
+
+            String versionTypeParam = ctx.queryParam(VERSION_TYPE);
```

Comment:
Why not use `ctx.ctx.queryParamAsClass()`?

Reply:
done